src/plugins.json

```diff
@@ -1248,5 +1248,11 @@
     "icon": "fork",
     "repository": "https://github.com/juanpicado/metalsmith-youtube",
     "description": "A Metalsmith plugin that lets you embed YouTube Videos."
+  },
+  {
+    "name": "metalsmith-hbt-md",
```

Comment:
The name field should be the human-friendly name of the plugin, not the NPM package name, and typically should not include the word "Metalsmith" as that's implied by the context.